src/bp/lang-server/service/index.ts

```diff
@@ -45,75 +54,172 @@ export default class LanguageService {
     return this._ready
   }
 
-  listModels(): AvailableModel[] {
-    return _.values(this._models)
+  listFastTextModels(): AvailableModel[] {
+    const allModels = _.values(this._models)
+    return allModels.map(m => m.fastTextModel)
   }
 
   private _discoverModels() {
     if (!fs.existsSync(this.langDir)) {
-      return []
+      return
     }
 
     const files = fs.readdirSync(this.langDir)
-    return files.forEach(f => {
+
+    type modelFileInfo = {
```

Comment:
We try to use interface when possible, and they should always start with an uppercase. Type is used when you need to edit its properties (eg: Pick<>, & , etc). Otherwise, interface is simpler 

As Maxime said, we try to keep all the definitions in the same file (or at the top of it if it is very specific), but never in a block of code

interface ModelFileInfo { }